### ISSUE TITLE
Weird comment indentation inside Closure

### ISSUE THREAD

**Issue by MicMonen** [phpfmt/sublime-phpfmt#195]
The comments get indented really far with spaces (normally tabs) inside the closure. The strange thing is when you change $self->id to e.g. 'lock-id' or other non-callable variable it will indent normally.

```
$self = $this;
$this->locker->lock($self->id, function () use ($self) {
	                         // comment 1
	                         // comment 2
	$self->method();
});
```

My troubleshooting information:
```
phpfmt (php version):
PHP 5.6.13 (cli) (built: Sep  9 2015 10:21:48)
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2015 Zend Technologies

phpfmt (php tokenizer) found

phpfmt (fmt.phar version):
/Users/Michel/Library/Application Support/Sublime Text 3/Packages/phpfmt/fmt.phar 14.1.0

PHP 5.5 compatibility mode: False
```

My Sublime Text 3 settings:
```
{
	"enable_auto_align": true,
	"excludes":
	[
		"PSR2IndentWithSpace"
	],
	"format_on_save": true,
	"indent_with_space": false,
	"passes":
	[
		"AddMissingCurlyBraces",
		"AddMissingParentheses",
		"AlignDoubleArrow",
		// "AlignGroupDoubleArrow",
		"AlignDoubleSlashComments",
		"AlignEquals",
		"ConvertOpenTagWithEcho",
		"DoubleToSingleQuote",
		"EliminateDuplicatedEmptyLines",
		"ExtraCommaInArray",
		"IndentTernaryConditions",
		"NormalizeLnAndLtrimLines",
		"PSR1OpenTags",
		"PSR2EmptyFunction",
		"ReindentAndAlignObjOps",
		"ReindentSwitchBlocks",
		"RemoveUseLeadingSlash",
		// "RTrim",
		"ShortArray",
		"SmartLnAfterCurlyOpen",
		"SpaceBetweenMethods",
		"StripNewlineAfterClassOpen",
		"StripNewlineAfterCurlyOpen"
	],
	"php_bin": "/usr/local/Cellar/php56/5.6.13/bin/php",
	"psr1": true,
	"psr2": true,
	"version": 2
}
```

**Comment by MicMonen** [comment]
AlignEquals seems to stopped working inside Closures also.

**Comment by ccirello** [comment]
open a new issue about that.

**Comment by ccirello** [comment]
upgrade to https://github.com/phpfmt/sublime-phpfmt/releases/tag/8.4.0

**Comment by ccirello** [comment]
> AlignEquals seems to stopped working inside Closures also.

Fixed in 8.5.0
